src/components/CounterBadge/index.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import {oneOf} from 'prop-types';
+import CoreBadge from 'wix-ui-core/Badge';
+import {ThemedComponent} from 'wix-ui-theme';
+
+import {textOfMaxTwoSymbols} from './content-validation';
+import {theme} from './theme';
+import UIText from '../UIText';
+import {SKIN} from './constants';
+
+const CounterBadge = ({skin, children}) => (
+  <ThemedComponent theme={theme} skin={skin}>
+    <CoreBadge>
+      <UIText appearance="T5" dataClass="badge-content">{children}</UIText>
+    </CoreBadge>
+  </ThemedComponent>
+);
+
+CounterBadge.propTypes = {
+  /** Type of the badge */
+  skin: oneOf(['default', 'standard', 'urgent', 'success']),
```

Comment:
You defined constants, so why not to use them here?

Reply:
I tried `Object.keys(SKIN)`, but as I mentioned, there are some weird issues, and as proptypes will be removed with typescript, I suggest not wasting time on those